### PR TITLE
OSDOCS-11836: Documenting the 4.13.49 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.13

Issue:
[OSDOCS-11836](https://issues.redhat.com/browse/OSDOCS-11836)

Link to docs preview:
[4.13.49](https://81168--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-49)

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -4325,3 +4325,28 @@ $ oc adm release info 4.13.48 --pullspecs
...
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.49 --pullspecs
+----
+
+[id="ocp-4-13-49-bug-fixes"]
+==== Bug fixes
```
Comment (dfitzmau) [inline release_notes/ocp-4-13-release-notes.adoc:4347]: No bugs so far. Check closer to release date.

**Review by sheriff-rh — APPROVED** [review]
LGTM; merging.

**Pull request merged**